generators/client-2/templates/src/main/webapp/app/account/_account.ng2module.ts

```diff
@@ -41,6 +49,12 @@ let ACCOUNT_STATES = [
     sessionsState,
     <%_ } _%>
     settingsState,
+<% if (enableSocialSignIn) { %>
```

Comment:
can you move this block above settingsState else when there is no socialLogin the settingsState will have a dangling coma